.nvidia-ci.yml

```diff
@@ -427,6 +461,8 @@ sign:ngc-ubuntu-rhel-rhcos:
     - .sign:ngc
   parallel:
     matrix:
+    - SIGN_JOB_NAME: ["ubuntu"]
+      VERSION: ["24.04"]
```

Comment:
missing DRIVER_VERSION

 - SIGN_JOB_NAME: ["ubuntu"]
      VERSION: ["24.04"]
      DRIVER_VERSION: ["565.57.01"]

Reply:
Removing this right now, as we we'll be adding the ngc release and sign jobs in another PR